lib/squasher/config.rb

```diff
@@ -68,18 +68,14 @@ def dbconfig
       return @dbconfig = nil unless File.exists?(dbconfig_file)
 
       begin
-        content = ERB.new(File.read(dbconfig_file)).result(binding)
-        @dbconfig = YAML.load(content)['development']
+        content = File.read(dbconfig_file).gsub(/database: (.+)/, 'database: squasher')
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.